ui/package.json

```diff
@@ -69,6 +69,7 @@
     "eslint-plugin-standard": "^5.0.0",
     "eslint-plugin-standard-typescript": "^1.0.1",
     "eslint-plugin-testing-library": "^5.6.0",
+    "intersection-observer": "^0.12.2",
```

Comment:
out of curiosity - what does this do?

Reply:
I needed to add this because it showed me an error when I did test that involved a modal (pop-up). Documentation says this:

`The Intersection Observer API allows you to configure a callback that is called when either of these circumstances occur: A target element intersects either the device's viewport or a specified element.`